.gitignore

```diff
@@ -10,7 +10,6 @@ yarn-error.log*
 # hardhat
 /artifacts
 /cache
-/typechain
```

Comment:
Since the package is published to npm, we shouldn't commit the typechain directory, but rather add it to the `files` section in `package.json`

Reply:
Okay, so I added the `files` field to `package.json`. It wasn't published that way so I just let it the way it was. Anyway, good 👍